.github/workflows/release-drafter.yml

```diff
@@ -2,36 +2,33 @@ name: Release Drafter
 
 on:
   push:
-    # our release branch
     branches:
-      - master
+      - '!*'
+    tags:
+      - 'v*'
 
 jobs:
   update_release_draft:
     runs-on: ubuntu-latest
     steps:
-      # Drafts your next Release notes as Pull Requests are merged into "master"
       - name: Drafting release
         id: release_drafter
         uses: release-drafter/release-drafter@v5
         with:
           config-name: release-drafter.yml
         env:
-          GITHUB_TOKEN: ${{ secrets.RELEASEDRAFTER_PAT }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Checkout release notes
-        if: github.event_name != 'pull_request' && startsWith(github.ref, 'refs/tags/') && success()
         uses: actions/checkout@v2
 
       - name: Write release notes to markdown file
         id: md_writer
-        if: github.event_name != 'pull_request' && startsWith(github.ref, 'refs/tags/') && success()
         run: |
-          printf '%b\n' "---\nlayout: release\ndate: $(date +'%Y-m-%d')\ntag: ${{ steps.release_drafter.outputs.tag_name }}\n---\n\n${{ steps.release_drafter.outputs.body }} " > ./docs/_releases/${{ steps.release_drafter.outputs.tag_name }}.md
+          printf '%b\n' "---\nlayout: release\ndate: $(date +'%Y-%m-%d')\ntag: ${{ steps.release_drafter.outputs.tag_name }}\n---\n\n${{ steps.release_drafter.outputs.body }} " > ./docs/_releases/${{ steps.release_drafter.outputs.tag_name }}.md
```

Comment:
A consideration:

Release Drafter runs each time a PR is merged, accumulating a trailing list of changes for the to-be-released version. With the current state of this workflow, does that mean that a new Automated Release Note PR will be created each time a PR is merged to `master` or would these merges result in an update to a pre-existing Automated Release Notes PR (one created each time a new draft release is created by Release Drafter)?


Optimization: Release Notes on docs.meshery.io could be p

Reply:
No, this workflow will not be triggered with PRs, it will be triggered with Github Releases (as I usually did before).

This [PR#2648](https://github.com/layer5io/meshery/pull/2648) was created only for testing purposes, validating that it will work when it should.

Also, I created that PR to have release notes for the latest version of Meshery (but we can close it if we want)

Reply:
Oh, very good. 🙂